### PR TITLE
First run import script

### PR DESCRIPTION
**Pull request by alxngyn**
fixes issue #125

## Changes in this PR.

- [X] Converted the client, projects and nodes data into a .csv file
- [X] Created a ruby script to import the data into the DB
- [X] ~~Modified app.rb to make the import script run~~
- [X] ~~Removed the "initially the model should be empty tests" since this will fail every time~~

## Testing this PR.

1. Do `docker-compose run --service-ports dev bash`
2. `rake migrate`
2. `ruby import/first_run_import.rb`
3. `rake spec`
4. `rake`
5. in a web browser go to `localhost:4567/clients`

OR
if you want to check the db:

5a. `sqlite3 /tmp/development.sqlite `
6. ` select * from clients;`
### Expected Output.

```
[root@5dc523949037 code]# rake migrate
Migrating to latest

[root@5dc523949037 code]# ruby import/first_run_import.rb 
Running first run import!
Finished running first run import

[root@5dc523949037 code]# sqlite3 /tmp/development.sqlite 
SQLite version 3.7.17 2013-05-20 00:56:22
Enter ".help" for instructions
Enter SQL statements terminated with a ";"

sqlite> .tables
clients         node_resources  plugins         projects        schema_info   

sqlite> select * from clients;
1|Drupal|bob|bob@example.com|bob loblaw
2|Rock the Vote|bob|bob@example.com|bob loblaw
3|Robot Operating System|bob|bob@example.com|bob loblaw
4|Apereo|bob|bob@example.com|bob loblaw
5|OSU Helpdesk|bob|bob@example.com|bob loblaw
6|Alembic|bob|bob@example.com|bob loblaw
7|Systers|bob|bob@example.com|bob loblaw
8|Teach Engineering|bob|bob@example.com|bob loblaw

== Sinatra (v1.4.7) has taken the stage on 4567 for development with backup from Thin
Thin web server (v1.7.0 codename Dunder Mifflin)
Maximum connections set to 1024
Listening on 0.0.0.0:4567, CTRL+C to stop

*should see clients/projects/nodes loaded*

```

@osuosl/devs

### CODE REVIEW

**Comment by subnomo** [comment]
Looks good. :+1:

**Comment by alxngyn** [comment]
Nevermind. This PR shouldn't have the OSL specific data in it. I'll need to figure out a way to make a independent script to load the data

File: import/first_run_import.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#
+# this file will import the csv data to the DB from the CSV only ONCE
+#
+require 'environment.rb'
```
Comment (ElijahCaine) [inline import/first_run_import.rb]: This should be `require_relative 'environment.rb'`

**Comment by ElijahCaine** [comment]
@alxngyn I'm not sure if you want to make that CSV data more agnostic, but it looks good to me.

- You need to change that `require` to a `require_relative`
- What happened to all of the styles? Is this going to merge like we think it will?

Other than that, lgtm. Pending plus on addressing that feedback.

**Comment by alxngyn** [comment]
@ElijahCaine fixed the require things. Not sure why it didn't show up before.

I didn't make changes to the styles so [I don't know](http://i3.kym-cdn.com/entries/icons/original/000/000/341/i-dunno-lol_1_.jpg)

**Comment by ElijahCaine** [comment]
LGTM +1

**Comment by hsolorza** [comment]
+1

**Pull request merged**